### PR TITLE
New task schedule jobs

### PR DESCRIPTION
**Pull request by mcalmer**
# Description

We have in the documentation a list of pre defined task jobs. This list was outdated and some jobs were missing.

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [x] Manager-4.3
- [ ] Manager-4.2

# Links

### CODE REVIEW

File: modules/administration/pages/task-schedules.adoc [hunk 1]
```diff
@@ -19,6 +19,9 @@ Clicking the start time links takes you back to the menu:Schedule Name[Basic Sch
 
 For example, the following predefined task bunches are scheduled by default and can be configured:
 
+menu:auto-errata-default:[]:
+Schedule auto errata updates as necessary
```
Comment (keichwa) [inline modules/administration/pages/task-schedules.adoc:23]:
```suggestion
menu:auto-errata-default:[]::
Schedule auto errata updates as necessary.
```

File: modules/administration/pages/task-schedules.adoc [hunk 2]
```diff
@@ -89,12 +95,20 @@ Synchronizes with SUSE Customer Center (SCC) (``mgr-sync-refresh``).
...
 Deletes stale client action data from the file system.
 First it tries to complete any possibly unfinished actions by looking up the corresponding results; these results are stored in the Salt job cache.
 An unfinished action can occur if the server has missed the results of the action.
 For successfully completed actions it removes artifacts such as executed script files.
 
+menu:minion-checkin-default:[]::
+Perform a regular check-in on minions.
+
+menu:notifications-cleanup-default:[]::
+Cleanup expired notification messages
```
Comment (keichwa) [inline modules/administration/pages/task-schedules.adoc:111]:
```suggestion
Cleanup expired notification messages.
```

File: modules/administration/pages/task-schedules.adoc [hunk 3]
```diff
@@ -118,5 +132,22 @@ Cleans up stale Web interface sessions, typically data that is temporarily store
 menu:ssh-push-default:[]::
 Prompts clients to check in with {productname} via SSH if they are configured with a `SSH Push` contact method.
 
+menu:system-profile-refresh-default:[]::
+Run a Hardware refresh on all systems. This happens only monthly and can increased load on the {productname} Server.
```
Comment (keichwa) [inline modules/administration/pages/task-schedules.adoc:136]:
```suggestion
Run a Hardware refresh on all systems. This happens only monthly and can increase load on the {productname} Server.
```

File: .changelog [hunk 4]
```diff
@@ -4,6 +4,7 @@
 # - Fixed error in Bat section of Upgrade Guide (bsc#1234567)
 # For guidelines: https://en.opensuse.org/openSUSE:Creating_a_changes_file_(RPM)
 
+- Add missing descriptions of taskomatic jobs
```
Comment (keichwa) [inline .changelog:7]:
```suggestion
- Added missing descriptions of taskomatic jobs in Administration Guide
```

**Review by keichwa — APPROVED** [review]
just cosmetic suggestions.

**Review by 0rnela — APPROVED** [review]

**Pull request merged**